### DISCUSSION TITLE
How to create config.json for VAE conversion to diffusers format?

### DISCUSSION THREAD

**Discussion by laetokang** [huggingface/diffusers#8963] [Q&A]
Hi ! I plan to fine-tune the SD v1.5 model using the data I've collected.
So, I downloaded the VAE from CivitAI.
However, there isn't a separate config.json file, and I'm curious about how I can create a config.json file

**Comment by a-r-r-o-w** [comment]
Hey! You could perform the conversion in two ways:
- Run [this](https://github.com/huggingface/diffusers/blob/4a782f462a063abbe3067d1944933576451bf6a9/scripts/convert_vae_pt_to_diffusers.py) script passing the `vae_pt_path` and `dump_path` to convert the vae to diffusers format. This will create a copy and use additional memory on your disk.
- You could use `AutoencoderKL.from_single_file("/path/to/vae.safetensors")`. [Relevant Docs](https://huggingface.co/docs/diffusers/v0.29.2/en/api/models/autoencoderkl#loading-from-the-original-format). This will not create a copy of the model in diffusers format on your disk, and simply load the other format and do the conversion in-memory each time.
